src/views/Registration/RegistrationDefineBusiness.vue

```diff
@@ -14,8 +14,8 @@
         <!-- Business Type -->
         <header id="business-type-confirm-header" />
         <BusinessTypeConfirm
-          class="py-8 px-6"
```

Comment:
This class is removed since div tag with "section-container" is used. See html above.

Reply:
This is added back in the last commit: fa7c46a.

Reply:
Vuetify helper classes "py-8 px-6" are the same as section-container style "padding: 2rem 1.5rem" so I'm not sure what "off by a couple pixels" refers to.

Reply:
Apparently, this is the same as before: padding: 32 24. We will see what UXA responds to this.